opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -189,12 +189,16 @@ namespace Opm {
                 ebosSimulator_.model().advanceTimeLevel();
             }
 
-            // set the timestep size and index in ebos explicitly
-            // we use our own time stepper.
-            ebosSimulator_.startNextEpisode( timer.currentStepLength() );
-            ebosSimulator_.setEpisodeIndex( timer.reportStepNum() );
-            ebosSimulator_.setTimeStepSize( timer.currentStepLength() );
-            ebosSimulator_.setTimeStepIndex( timer.reportStepNum() );
+            // set the timestep size and episode index for ebos explicitly. ebos needs to
+            // know the report step/episode index because of timing dependend data
+            // despide the fact that flow uses its own time stepper. (The length of the
+            // episode does not matter, though.)
+            Scalar t = timer.simulationTimeElapsed();
+            ebosSimulator_.startNextEpisode(/*episodeStartTime=*/t, /*episodeLength=*/1e30);
```

Comment:
This line looks like a clear bugfix to me. The rest is more about adding the information expected for the vtk output, correct?

Reply:
the real bugfix is the `setTime()` call bellow. starting a new episode is only necessary so that flow's and eWom's time management code don't clash. (eWoms makes sure to align time steps with the episode boundaries, i.e. if it thinks that an episode will be over after a time step, it will chop the time step and this may lead to NaNs.)

> The rest is more about adding the information expected for the vtk output, correct?

not really: the actual VTK output works even without this fix, but the times in the "meta file" that binds the time step files together will be non-ascending which is (a) wrong and (b) confuses the hell out of paraview...